tests/src/test/resources/infinispan/hono-data-grid.xml

```diff
@@ -1,43 +1,33 @@
 <infinispan
         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-        xsi:schemaLocation="urn:infinispan:config:11.0 https://infinispan.org/schemas/infinispan-config-11.0.xsd
-                            urn:infinispan:server:11.0 https://infinispan.org/schemas/infinispan-server-11.0.xsd"
-        xmlns="urn:infinispan:config:11.0"
-        xmlns:server="urn:infinispan:server:11.0">
+        xsi:schemaLocation="urn:infinispan:config:13.0 https://infinispan.org/schemas/infinispan-config-13.0.xsd
+                            urn:infinispan:server:13.0 https://infinispan.org/schemas/infinispan-server-13.0.xsd"
+        xmlns="urn:infinispan:config:13.0"
+        xmlns:server="urn:infinispan:server:13.0">
 
   <cache-container name="routing-info">
     <security>
       <authorization>
-        <identity-role-mapper /> 
-        <role name="admin" permissions="ALL" /> 
-        <role name="adapter" permissions="READ WRITE BULK_READ" />
+        <identity-role-mapper />
+        <roles>
+          <role name="admin" permissions="ALL" /> 
+          <role name="adapter" permissions="READ WRITE BULK_READ" />
+        </roles>
       </authorization>
     </security>
-    <local-cache name="command-router">
-      <encoding>
-        <key media-type="text/plain" />
-        <value media-type="text/plain" />
-      </encoding>
-      <persistence passivation="false">
-        <file-store
-            shared="false"
-            preload="true"
-            fetch-state="false"
-            read-only="false"
-            purge="false"
-            path="command-router">
-        </file-store>
-      </persistence>
+    <local-cache name="command-router" simple-cache="true">
+      <encoding media-type="application/x-protostream" />
+      <memory max-count="200" storage="OFF_HEAP" when-full="REMOVE"></memory>
       <security>
         <authorization enabled="true" />
       </security>
     </local-cache>
   </cache-container>
 
-  <server xmlns="urn:infinispan:server:11.0">
+  <server xmlns="urn:infinispan:server:13.0">
     <interfaces>
       <interface name="public">
-        <non-loopback />
+        <server:inet-address value="0.0.0.0" />
```

Comment:
nit: The `server` prefix looks superfluous here (`<server>` tree already having the corresponding namespace).